docusaurus.config.js

```diff
@@ -70,6 +71,7 @@ module.exports = {
                     // editCurrentVersion: true,
                     // onlyIncludeVersions: process.env.PREVIEW_DEPLOY === "true" ? ["current", ...versions.slice(0, 2)] : undefined,
                 },
+                ...(gtmConfig["containerId"] && { googleTagManager: gtmConfig }),
                 ...(gtagConfig["trackingID"] && { gtag: gtagConfig }),
```

Comment:
Let us preserve the alphabetical order:

```suggestion
                ...(gtagConfig["trackingID"] && { gtag: gtagConfig }),
                ...(gtmConfig["containerId"] && { googleTagManager: gtmConfig }),
```